### ISSUE TITLE
[BE] [SPIKE] For Travel Claims Worker, handle the 60 second limit for Sidekiq worker

### ISSUE THREAD

**Issue by randomsync** [department-of-veterans-affairs/va.gov-team#64070]
## User Story
As a Veteran, I want to be able to submit a travel claim as part of online check-in

## Tasks
- [ ] [BE] Research how we can handle the 60 second limit on Sidekiq worker execution time and propose options

## Acceptance Criteria
- [ ] Create implementation ticket(s)

**Comment by loripusey** [comment]
@randomsync can you point this before planning today? thanks

**Comment by randomsync** [comment]
As part of the spike, and with support from platform, we realized that the timeout is coming from the Net::HTTP ruby library that Faraday uses. Since we don't use platform's base library for Travel Claims code (could be an enhancement ticket later on), the best way to handle this was to increase the timeout when we make the call to BTSSS to submit the claim. I have opened a PR to increase the timeout to 120 seconds.

We still need to handle the timeout error appropriately and text Veterans with the generic error message that we send to them in case of other errors.
